src/main/java/stsjorbsmod/cards/cull/ReapAndSow.java

```diff
@@ -0,0 +1,61 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.IncreaseManifestAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.powers.OverkillPower;
+import stsjorbsmod.powers.ReapAndSowPower;
+
+public class ReapAndSow extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(ReapAndSow.class);
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 2;
+    private static final int DAMAGE = 25;
+    private static final int UPGRADE_DAMAGE = 8;
+
+    private int overkill = 0;
+
+    public ReapAndSow() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = damage = DAMAGE;
+        magicNumber = baseMagicNumber = 0;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        overkill = getOverKill(m);
+        if (overkill > 0 && m.hasPower("Minion")) {
+            addToBot(new ApplyPowerAction(p, p, new ReapAndSowPower(p, overkill)));
+        }
+        addToBot(new DamageAction(m, new DamageInfo(p, damage), AbstractGameAction.AttackEffect.SLASH_DIAGONAL));
+    }
+
+    private int getOverKill(AbstractMonster mo) {
+        super.calculateCardDamage(mo);
+        if (this.damage > mo.currentHealth) {
+            this.overkill = this.damage - mo.currentHealth;
```

Comment:
I think this is probably a little too simplified; for example, I don't think it's going to do the right thing against enemies with block or vulnerable.

I think the best approach here would probably be calculate this difference after all the damage-modifying effects are applied in `AbstractMonster.damage`; the `lastDamageTaken` field that method sets is *almost* exactly what we want, if only it didn't get clamped to the monster's current help. Since we don't want to rewrite all the calculation logic and the values we need are lost by the time `AbstractMonster.damage` is done, I'm not sure there's a better option than adding a patch here. I'd probably lean towards adding an `AbstractMonster` patch which:
* Adds a `SpireField` to `AbstractMonster` for `lastOverkillDamage`, which defaults to 0
* Adds an `Instrument` patch for `AbstractMonster.damage` that replaces the `this.lastDamageTaken` field access with `$_ = $proceed(); lastOverkillDamageField.set(__this, damageAmount - __this.lastDamageTaken);`